wpilibj/src/athena/java/edu/wpi/first/wpilibj/hal/HAL.java

```diff
@@ -16,7 +16,7 @@
 public class HAL extends JNIWrapper {
   public static native void waitForDSData();
 
-  public static native int initialize(int mode);
+  public static native int initialize(int timeout, int mode);
```

Comment:
I think I do want to leave this an int. That way if we ever need to add more error messages, its possible. Now that the error returns are standard (0 means true) that should be easier.